src/themes/basic/_layout.styl

```diff
@@ -277,8 +278,9 @@ main.hidden
   transition opacity 0.3s
   width $sidebar-width - 16px
   z-index 30
+  cursor pointer
 
-  .sidebar-toggle-button:hover
+  &:hover .sidebar-toggle-button
```

Comment:
This is not required I guess....

Reply:
![image](https://user-images.githubusercontent.com/44045911/75405813-b4dba680-5949-11ea-84aa-f51587458b0b.png)

The sidebar is toggled on clicking `.sidebar-toggle`, it may be confusing if it only changes opacity when you hover `.sidebar-toggle-button`.

Reply:
can you please explain this a bit. Like how the current styling is working and how your changes will work 

THanks

Reply:
| Before | After |
| ------ | ----- |
|![Gh4ZsWQe0l](https://user-images.githubusercontent.com/44045911/75419076-fd0dbf80-596f-11ea-9593-e0e869c6f45f.gif)|![TbMXodpPwA](https://user-images.githubusercontent.com/44045911/75419384-b10f4a80-5970-11ea-81de-0ab75e771cbf.gif)|